android/build.gradle

```diff
@@ -39,7 +39,7 @@ buildscript {
             ? rootProject.ext['kotlinVersion']
             : rootProject.hasProperty('AsyncStorage_kotlinVersion')
             ? rootProject.properties['AsyncStorage_kotlinVersion']
-            : '1.5.31'
+            : '1.6.10'
```

Comment:
Can I ask you to do a small housekeeping too, please?

Please drop this option, as since [Kotlin 1.5, the default target is 1.8](https://kotlinlang.org/docs/whatsnew15.html#new-default-jvm-target-1-8)
```diff
-        kotlinOptions {
-            jvmTarget = "1.8"
-        }
```

Please also update values:
```diff
-    compileSdkVersion safeExtGet('compileSdkVersion', 28)
+    compileSdkVersion safeExtGet('compileSdkVersion', 31)
    defaultConfig {
-        minSdkVersion safeExtGet('minSdkVersion', 19)
+        minSdkVersion safeExtGet('minSdkVersion', 21)
-        targetSdkVersion safeExtGet('targetSdkVersion', 28)
+        targetSdkVersion safeExtGet('targetSdkVersion', 29)
    }
```

Reply:
Many thanks if you do! 🙏

Reply:
Sure, already done :)